eng/Version.Details.xml

```diff
@@ -2,93 +2,89 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.VisualStudio.LanguageServices" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.VisualStudio.LanguageServices" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.CodeAnalysis.Test.Utilities" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.Test.Utilities" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.Features" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.Features" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.EditorFeatures" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.EditorFeatures" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.CodeAnalysis.LanguageServer.Protocol" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.LanguageServer.Protocol" Version="5.0.0-1.25230.6">
+    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.FSharp" Version="5.0.0-1.25274.2">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.FSharp" Version="5.0.0-1.25230.6">
-      <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
     </Dependency>
     <Dependency Name="Microsoft.CodeAnalysis.Analyzers" Version="3.12.0-beta1.25230.6">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>ded867328249b5a9b9e6e29e3f07abc19111f5d1</Sha>
+      <Sha>392fbb66a16ca3d34abdfa93cc5a3717f447d25e</Sha>
```

Comment:
This one always trips me up too, it builds from the same commit, but as a different version, so its easy to accidentally update the sha without bumping the package if you just find-and-replace (which, to be clear, is exactly what I do too 😁)

Reply:
Argh, find and replace got me again!